_apps/tap-release.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Tap Release
+description: Update Homebrew taps when you publish new GitHub releases.
+slug: tap-release
+screenshots:
+- https://raw.githubusercontent.com/toolmantim/tap-release/master/design/screenshot.png
+authors: [ toolmantim ]
+repository: toolmantim/tap-release
+host: https://tap-release.now.sh
+---
+
+## Usage
+
+Firstly, you’ll need to install the [Tap Release GitHub App](https://github.com/apps/tap-release). This listens out for any releases, or any changes to the configuration.
+
+Then, add a `.github/tap-release.yml` configuration file to the GitHub repository where you publish new releases to.
+
+For example, given the following `.github/tap-release.yml` file in a `my-org/app/app.rb` repository:
```

Comment:
> `my-org/app/app.rb` repository

I see further down the importance of the app.rb file, but the usual syntax for referring to a repo is just owner/repo, so maybe that would be more clear here?

Reply:
Oh, that was a straight up typo! Thanks for the catch. All fixed!